addons/hw_drivers/iot_handlers/drivers/DisplayDriver_L.py

```diff
@@ -173,3 +217,42 @@ def get_iot_devices(self):
         } for device in iot_devices]
 
         return json.dumps({'iot_device_status': iot_device})
+
+    @http.route('/kiosk/display', type='json', auth='none', methods=['POST'])
```

Comment:
why do you need a new controller for this ? Why not simply point the kiosk to the regular address of the self order page ?